android/app/src/main/kotlin/com/jabook/app/jabook/audio/data/repository/PlaybackPositionRepository.kt

```
@@ -58,6 +59,7 @@ public class PlaybackPositionRepository
                 positionDao.upsertPosition(entity)
                 Result.Success(Unit)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)